cmd/rigs/cmd/publish_renders.go

```diff
@@ -18,10 +21,30 @@ var rendersCmd = &cobra.Command{
 	Short: "Publish rig renders to nft.storage",
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := cmd.Context()
+
 		path := viper.GetString("renders-path")
-		cid, err := dirPublisher.PublishDir(ctx, path)
-		checkErr(err)
-		checkErr(localStore.TrackCid(ctx, "renders", cid))
-		fmt.Printf("Renders published with cid %s\n", cid)
+		cidString := viper.GetString("cid")
+		var c cid.Cid
+		var err error
+		if cidString != "" {
+			c, err = cid.Decode(cidString)
+			checkErr(err)
+		}
+		chunksDir := viper.GetString("chunks-dir")
+
+		if chunksDir == "" && c.String() == "" {
+			c, err = dirPublisher.DirToIpfs(ctx, path)
+			checkErr(err)
+			fmt.Printf("Images added to IPFS with cid %s\n", c.String())
+		}
+		if chunksDir == "" && c.String() != "" {
+			chunksDir, err = dirPublisher.CidToCarChunks(ctx, c)
+			checkErr(err)
+			fmt.Printf("Car chunks written to folder %s\n", chunksDir)
+		}
+		checkErr(dirPublisher.CarChunksToNftStorage(ctx, chunksDir))
+		fmt.Printf("Renders published with cid %s\n", c.String())
+
+		checkErr(localStore.TrackCid(ctx, "renders", c.String()))
```

Comment:
Same thing here. Just making image publishing "resumable" in case of failures/errors along the way since it takes a long time to publish 3000 Rig images.